dezoomify.py

```diff
@@ -214,12 +214,15 @@ def __init__(self, args):
                 self.zoom_level = zoom_level # Reinitialize zoom level between each download
                 destination = self.out_names[i]
                 self.log.info("[{}/{}] Processing image {}...".format(i + 1, len(self.image_urls), image_url))
+                any_exceptions = False
                 try:
                     self.process_image(image_url, destination)
                 except Exception as e:
+                    any_exceptions = True
```

Comment:
It would be simpler to move the message *Dezoomified image created [...]* to the `try` block after line 221. Thus, you wouldn't need to add the variable `any_exception`.